games.rb

```diff
@@ -1,5 +1,53 @@
 class Games 
   def balls(bowling)
-    @bowling = bowling
+    counter_frame = 0
+    ball = 0
+    frame = []
+    score = 0
+    while ball < 19
+      total = game_rules(bowling[ball], bowling[ball + 1], bowling[ball + 2], bowling[ball + 3], bowling[ball + 4], counter_frame)
+      score += total
+      frame[counter_frame] = score
+      counter_frame += 1
+      ball += 2
+    end
+    return frame
+  end
+
+  def game_rules(first_ball, second_ball, third_ball, fourth_ball, fifth_ball, counter_frame)
```

Comment:
mas de tres parametros en un metodo no es una buena practica, podrias usar un array para los `xxxx_ball` o un hash, y asi solo tendrias mandar dos argumentos cada que llamas tu funcion, por ejemplo: 
```suggestion
  def game_rules(balls, counter_frame)
```
en caso de ser un hash, puede ser algo asi: 
`balls = { first: 0, second: 0, third: 0, fourth: 0, fifth: 0 }`
o array:
`balls = [0, 0, 0, 0, 0]`